packages/components/src/summary/number.js

```diff
@@ -48,10 +47,9 @@ const SummaryNumber = ( {
 	selected,
 	value,
 	onLinkClickCallback,
-	isHomescreen,
 } ) => {
 	const liClasses = classnames( 'woocommerce-summary__item-container', {
-		'is-homescreen': isHomescreen,
+		'is-homescreen': true,
```

Comment:
This is an artefact from when Summary Numbers differed between on Reports and the Homescreen. I just double checked and this CSS class name doesn't appear to have an impact if you remove `:not(.is-homescreen)` from this line.

https://github.com/woocommerce/woocommerce-admin/blob/6198ccf165e46530caf8040aa2162a5d2e6aa904/packages/components/src/summary/style.scss#L202

So, I think this logic can be removed completely, including the `:not(.is-homescreen)`.

Reply:
Nice, done in 4ac8ded